Automattic/wp-calypso
# Migration Flow: call Guides endpoint on step change

## Proposed Changes

Calls the `triggerGuidesForStep` function on step change in the Migration flow.

## Testing Instructions

- Go to  `/setup/site-migration`
- You may be redirected to start/domains. Select "Choose domain later"
- Go through the flow to select Free plan
- When you get to "What are your goals?" 
- Open the browser network log.
- Select "Import Existing Content or Website"
- When you get to `setup/site-migration` confirm that a network request is sent to `guides/trigger` endpoint.

## Pre-merge Checklist
